index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/136dc91701.js" crossorigin="anonymous"></script>
+    <title>CC Global summit 2015</title>
+</head>
+<body>
+    <header>
+        <div>
+            <nav>
+             <ul class="headerTop">
+                <li class="fb"><a href=""><i class="fa-brands fa-facebook-f"></i></a></li>
+                <li class="twit"><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                <li class="eng"><a href="">English</a></li>
+                <li class="pg"><a href="">My Page</a></li>
+                <li class="ld"><a href="">Logout</a></li>
+             </ul>
+            </nav>
+        </div>
+        <div class="headerBtm">
+            <div class="logoSite">
+                <img class="logo" src="./icon/gentle.jpeg" alt="Logo" onclick="fun()">
+                <b>Ideal <br> Men</b>
+            </div>
+            <nav>
+            <ul  class="mainNavBar">
+                 <li class="about"><a href="">About</a></li>
+                 <li><a href="">Program</a></li>
+                 <li><a href="">Join</a></li>
+                 <li><a href="">Sponsor</a></li>
+                 <li><a href="">News</a></li>
+                 <li class="compain"><a href="">CC Campain</a></li>
+            </ul>
+
+            </nav>
+        </div>
+    </header>
+    <nav class="nav_bar" id="nav_bar">
+        <ul id="nav_ul">
+            <li><a href="./index.html" >Home</a></li>
+        <li class="about"><a href="./mobile.html">About</a></li>
+        <li><a href="#program">Program</a></li>
+        </ul>
+    </nav>  
+    <main class="principal">
+        <div>
+            <i id="menu-icon" class="fa-solid fa-bars" onclick="showNav()"></i>
```

Comment:
- [ ] Great job so far, However, I see that after using the navbar on a mobile device when I change it to a desktop device the hamburger menu still appears. Kindly fix that.

![image](https://user-images.githubusercontent.com/59416136/163624700-1833f77c-b6d9-4dc7-bba8-4f2ce644951f.png)